# disables use of pending statement in the test suite The required gem,
# pending, is not packaged in Fedora and since the statement is merely a task
# note, it's safe to disable it's usage for the purpose of packaging.
diff --git a/test/test_helper.rb b/test/test_helper.rb
index 3b27218..964ff59 100644
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -10,7 +10,7 @@ rescue LoadError
 end
 require 'htmlentities'
 require 'nokogiri'
-require 'pending'
+#require 'pending'
 
 ENV['SUPPRESS_DEBUG'] ||= 'true'
 
